src/components/results/candidate-results/personality-details/index.js

```diff
@@ -36,17 +36,17 @@ class PersonalityDetails extends Component {
     if(settings.length === 0) { disableSettings = true; }
     if(tools.length === 0) { disableTools = true; }
     if(disableSettings && disableTools) { return null; }
-    let detailClass;
+    let detailColumn;
     [!disableTools, perspective === "thirdPerson", !disableSettings].filter(Boolean).length > 2 ? (
-      detailClass = style.detail
+      detailColumn = style.detail
     ) : (
-      detailClass = style.detail6Columns
+      detailColumn = style.detail6Columns
```

Comment:
Sorry I meant changing the name of the style. `const detailClass = style.detailColumn;`